src/recordlinker/linking/link.py

```diff
@@ -113,36 +114,58 @@ def link_record_against_mpi(
                                 matched_count += 1
```

Comment:
Do we plan on logging any of the feature comparison results (e.g., similarity score, log odds score, log odds ratio, fuzzy threshold), as indicated in #131?

I think it would be helpful for explainability purposes (why was this predicted a `match` or not?). I found myself often logging these results manually while working on past tickets to better understand why a Patient record was predicted a `match` or not.

Reply:
Let me add something now just so we have it.  We can run some tests, and if its still too much noise we can downgrade it to a debug statements so its not always present.

Reply:
I added some logging to the compare function for this

https://github.com/CDCgov/RecordLinker/pull/135/files#diff-7bc8ec967134462397574bcf3aa1c2be6e37f22c69ce36cbb36fbcbce8b1136cR64